src/atoms/LinkBarLinkBase.js

```diff
@@ -43,21 +43,24 @@ export const LinkBarLinkBase = styled.a`
 	border: 0;
 	outline: none;
 	text-decoration: none;
-	color: ${getTextColor} !important;
 	font-family: ${props => props.theme.variables.headingsFont};
 	font-size: ${props => props.theme.variables.uiRegularSize};
 	line-height: ${props => (props.isBlockLink ? '0' : props.theme.variables.uiRegularLineHeight)};
 	font-weight: ${props => (props.isActive ? '600' : '400')};
 	transition: padding .2s;
 	background: ${props => (props.isActive ? props.activeBackground : 'transparent')};
-
+	text-transform: ${props => (props.uppercase && 'uppercase')};
```

Comment:
Elsewhere in shiny we use the prop `ALLCAPS` - it would be nice if we could stay consistent. Also, this line will produce borked css if the prop isn't set